jupyter_ydoc/ydoc.py

```diff
@@ -10,87 +10,185 @@
 
 class YBaseDoc(ABC):
     """
-    Base class
+    Base YDoc class.
+    This class, defines the minimum API that any documents must provide
+    for the :class:`YDocWebSocketHandler` to be able to get and set the
```

Comment:
`jupyter_ydoc` is agnostic to the provider, I don't think we should mention WebSockets.